### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8916,6 +8925,8 @@ doccumented->documented
 doccuments->documents
 dockson->dachshund
 docmenetation->documentation
+docmunet-document
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8928]: Fix the typo with the arrow please.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2529,6 +2529,9 @@ attributei->attribute
 attributen->attribute
 attributred->attributed
 attributs->attributes
+attriebute->attribute
+attriebuted->attributed
+attriebutes->attributes
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:2534]: Add attriebuting->attributing
```suggestion
attriebutes->attributes
attriebuting->attributing
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -8916,6 +8925,8 @@ doccumented->documented
 doccuments->documents
 dockson->dachshund
 docmenetation->documentation
+docmunet-document
+docmunets->documents
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8929]: Add some more variants
```suggestion
docmunetation->documentation
docmuneted->documented
docmuneting->documenting
docmunets->documents
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -19254,6 +19268,7 @@ propertery->property
 propertie->property, properties,
 propertise->properties
 propertly->properly, property,
+propertu->property
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:19271]:
```suggestion
propertu->property
propertus->properties
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -24603,6 +24624,8 @@ tipically->typically
 titel->title
 titels->titles
 titile->title
+tittle->title
+tittles->titles
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24628]:
```suggestion
tittled->titled
tittles->titles
tittling->titling
```

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -21846,6 +21861,8 @@ roduceer->producer
 roomate->roommate
 ropeat->repeat
 rorated->rotated
+rosponse->response
+rosponsive-responsive
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:21865]: Again with the arrow!

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -24918,6 +24944,10 @@ tipically->typically
 titel->title
 titels->titles
 titile->title
+tittle->title
+tittled->titled
+tittles->titles
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24949]: It seems tittle and tittles should be in dictionary-rare please.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -26117,6 +26148,7 @@ unxpected->unexpected
 unziped->unzipped
 upadate->update
 upadated->updated
+upadating->updating
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:26151]: Add updater and updates variant

```suggestion
upadater->updater
upadates->updates
upadating->updating
```

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Done 0c85d78

**Comment by luzpaz** [comment]
Rebased and ready for merge

**Comment by peternewman** [comment]
Can you add `desogners->designers` in #1492 or a new PR please.

**Pull request merged**